Misc/NEWS.d/next/Build/2021-02-02-16-26-44.bpo-43103.VWeyP_.rst

```diff
@@ -0,0 +1,3 @@
+Add configure ``--without-static-libpython`` option to not build
```

Comment:
I'm still confused, what about:

> Add configure ``--with/without-static-libpython`` option to not build
``libpythonMAJOR.MINOR.a`` static library and not install ``python.o`` object
file. Defaults to ``--without-static-libpython``.